openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -343,6 +343,75 @@ Note: The issuer MAY decide to support both options. In which case, it is at the
 
 A Credential Format Profile for Credentials complying with IETF SD-JWT VCs [@!I-D.ietf-oauth-sd-jwt-vc] is defined in Annex A.3 of [@!OIDF.OID4VCI] and Annex A.4 of [@!OIDF.OID4VP].
 
+## SD-JWT VC Data Model (SD-JWT VCDM)
+
+SD-JWT VCDM is a data model that follows IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc], but allows the usage of [@!W3C.VCDM1.1] or [@!W3C.VCDM2.0]. When IETF SD-JWT VC is mentioned in this specification, SD-JWT VCDM define in this section MAY be used.
+
+For backward compatibility with JWT processors, the following registered JWT claims MUST be used, instead of their respective counterpart properties in [@!W3C.VCDM1.1] or [@!W3C.VCDM2.0]:
+
+* `exp` Claim MUST represent the `expirationDate` property, encoded as a UNIX timestamp (NumericDate).
+* `iss` Claim MUST represent the `issuer` property.
+* `iat` Claim MUST represent `issuanceDate` property, encoded as a UNIX timestamp (NumericDate).
+* `status` Claim MUST represent `credentialStatus` property.
+* `schema` Claim MUST represent the `credentialSchema` property.
+* `sub` Claim MUST represent the `id` property of `credentialSubject` property.
+
+IETF SD-JWT VC is extended with the following claims:
+
+* `vcdm`: OPTIONAL. Contains properties defined in [@!W3C.VCDM1.1] or [@!W3C.VCDM2.0] that are not represented by their counterpart JWT Claims as defined above.
+
+The following is a non-normative example of an unsecured payload of an SD-JWT VCDM, that is built using the example of unsecured payload in Section 3.3 of [@!I-D.ietf-oauth-sd-jwt-vc]:
```

Comment:
expectation is for all user claims being included in vcdm, not outside of it. it makes more sense to define a static vct that is used when vcdm claim is present. and that static vct can have a flag that json-ld processing applies to vcdm. but this might be too complex

Reply:
How is sd-jwt vc defining "namespaces"/"vocabularies"?
Namely, all the issues raised here are due to lack of it.

Reply:
> How is sd-jwt vc defining "namespaces"/"vocabularies"?

sd-jwt vc has invented its own way of doing this, called "JWT VC Type Metadata"

Reply:
Interesting. How do you (unambiguously) represent a claim from:
a) IANA (any IANA registered claim, not only JWT registered claim)
b) claim registered in OID? (used by x509 and mDOC/mdl)
c) public vocabulary, such as ELM
d) private vocabulary

Reply:
With the current proposal, everything outside the `vcdm` claim would be IANA/standard JWT claims; everything inside would be the same as in W3C VCDM.

Reply:
@awoie and @danielfett , I would kindly ask you to answer the question:

How do you (unambiguously) represent a claim from:
a) IANA (any IANA registered claim, not only JWT registered claim)
b) claim registered in OID? (used by x509 and mDOC/mdl)
c) public vocabulary, such as ELM
d) private vocabulary

Note: JSON-LD is not mentioned anywhere. In my view, as expressed several times, JSON-LD is not posing any issues at all, if positioned correctly. (as JSON-LD extends JSON with additional properties, it's very easy to model the solution with clear boundaries)